community/community_support_process.md

```diff
@@ -0,0 +1,25 @@
+# Community Support Process
+
```

Comment:
OpenEBS supports the community via [GitHub issues](https://github.com/openebs/openebs/issues) or through Kubernetes Slack. For details on joining the slack, please see the instructions [here](https://github.com/openebs/openebs/tree/master/community#contact).